index.js

```diff
@@ -87,32 +97,47 @@ function createClubhouseStoryForGithubIssue(issue) {
   let url = issue.html_url
   let id = issue.id
 
-  return ch.createStory({
+  let story = {
     name: title,
     description: body,
     project_id: TILT_PROJECT_ID,
-    labels: [{name: 'exterminator'}],
-    external_tickets: [{external_id: String(id), external_url: url}]
-  })
-}
+    labels: [{ name: "exterminator" }],
+    external_tickets: [{ external_id: String(id), external_url: url }],
+  }
 
-octokit.issues.get({
-  owner: 'windmilleng',
-  repo: 'tilt',
-  issue_number: issueNumber,
-}).then((response) => {
-  let issue = response.data
-  let storyPromise = findClubhouseStoryForGithubIssue(issue)
-  return Promise.all([issue, storyPromise])
-}).then(([issue, existingStory]) => {
-  if (existingStory) {
-    console.log('Found existing clubhouse issue:\n' + existingStory.app_url)
-  } else {
-    return createClubhouseStoryForGithubIssue(issue).then((response) => {
-      console.log('Created new clubhouse issue:\n' + response.app_url)
-    })
+  if (isDryRun) {
+    return new Promise((resolve, _) => resolve(story))
   }
-}).catch((err) => {
-  console.error(err)
-  process.exit(1)
-})
+
+  return ch.createStory(story)
+}
+
+octokit.issues
+  .get({
+    owner: "windmilleng",
+    repo: "tilt",
+    issue_number: issueNumber,
+  })
+  .then(response => {
+    let issue = response.data
+    let storyPromise = findClubhouseStoryForGithubIssue(issue)
+    return Promise.all([issue, storyPromise])
+  })
+  .then(([issue, existingStory]) => {
+    if (existingStory) {
+      console.log("Found existing clubhouse issue:\n" + existingStory.app_url)
+    } else {
+      return createClubhouseStoryForGithubIssue(issue).then(response => {
+        if (isDryRun) {
+          console.log("Would create new clubhouse issue:\n")
```

Comment:
consider setting this line apart visually and/or adding another line below to the effect of "🤐no changes made" so it's obvious at a glance whether we changed state or not. (Visually this looks v. similar to the "create an issue" message.)